frontend/pages/hosts/details/cards/Software/InstallStatusCell/InstallStatusCell.tsx

```diff
@@ -44,11 +45,27 @@ export const INSTALL_STATUS_DISPLAY_OPTIONS: Record<
       </>
     ),
   },
+  verifying: {
+    iconName: "success-outline",
+    displayText: "Verifying",
+    tooltip: () => (
+      // TODO
+      <>TODO InstallStatusCell.tsx</>
```

Comment:
?

Reply:
same, this is host details page, which is your ticket